doc/conf.py

```diff
@@ -106,6 +109,11 @@
     "announcement": "Check out the Free "
     " <a href='https://www.ntnu.edu/web/temgemini/nordtemhub'>(Big) Data analysis of 4D-STEM</a> "
     " workshop in Trondheim, Norway, 11-13 June 2024",
+    "switcher": {
+        # Update when merged and released
```

Comment:
This comment is a bit misleading because it sounds like that something needs to be updated here and not in the `json` file itself.

Reply:
Changed that thanks!